src/work-allocation/resolvers/task.resolver.ts

```diff
@@ -1,25 +1,30 @@
 import { Injectable } from '@angular/core';
 import { ActivatedRouteSnapshot, Resolve, Router, RouterStateSnapshot } from '@angular/router';
-import { EMPTY, Observable } from 'rxjs';
+import { combineLatest, EMPTY, Observable } from 'rxjs';
 import { catchError } from 'rxjs/operators';
+import { Caseworker } from '../models/dtos';
 
 import { Task } from '../models/tasks';
-import { WorkAllocationTaskService } from '../services';
+import { CaseworkerDataService, WorkAllocationTaskService } from '../services';
 import { handleFatalErrors, WILDCARD_SERVICE_DOWN } from '../utils';
 
 @Injectable({ providedIn: 'root' })
-export class TaskResolver implements Resolve<Task> {
+export class TaskResolver implements Resolve<any> {
```

Comment:
The same comment about the `any` as the one below...

Reply:
Agree, will change it to object type

Reply:
Done

Reply:
This still shows as `any` rather than `{ task: Task; caseworkers: Caseworker[]; }`...?